### PR TITLE
Add question : Does your IT organization have a formal and/or sponsored initiatives in place

### PR DESCRIPTION
**Pull request by dgarros**
The question was initially proposed in #61 by @jvanderaa and @abenokraitis

I tried to combine both proposals, I hope I captured your ideas correctly

### CODE REVIEW

**Comment by abenokraitis** [comment]
maybe update option 3 from:
` - No, the leadership is not supportive of our initiative add more automate (neutral)`
to
` - No, our leadership is not formally investing in automation, but I am to get my job done (neutral)`

**Comment by dgarros** [comment]
@abenokraitis  I've updated the 3rd option as proposed, thanks

File: questionnaire/2020/questions.yaml [hunk 1]
```diff
@@ -285,6 +285,15 @@
     - 2 to 5 years 
     - More than 5 years
 
+  - title: Does your IT organization have a formal and/or sponsored initiatives in place to expand and improve the use of automation on the network?
```
Comment (fracaen) [inline questionnaire/2020/questions.yaml]: Remove the "a" in "Does your IT organization have a formal and/or sponsored initiatives", so everything is plural.

File: questionnaire/2020/questions.yaml [hunk 2]
```diff
@@ -285,6 +285,15 @@
...
+    type: Single choice 
+    id: leadership-support
+    responses:
+    - Yes, this is critical to the top-level IT strategy (proactive)
```
Comment (fracaen) [inline questionnaire/2020/questions.yaml]: For consistency with the following answers: "Yes, this is critical to our IT leadership's strategy"

**Review by fracaen — APPROVED** [review]

**Pull request merged**